--- Python-3.4.1/Lib/test/test_gdb.py.153~	2014-06-12 16:33:44.342930073 +0200
+++ Python-3.4.1/Lib/test/test_gdb.py	2014-06-12 16:36:53.556564577 +0200
@@ -182,6 +182,9 @@ class DebuggerTests(unittest.TestCase):
             # Issue #19753: missing symbols on System Z
             'Missing separate debuginfo for ',
             'Try: zypper install -C ',
+	    'warning: Unable to open',
+	    'Try: yum --disablerepo=',
+	    'Undefined set print command'
             )
         for line in errlines:
             if not line.startswith(ignore_patterns):
