diff -Naur Python-3.5.3/setup.py Python-3.5.3.tpg/setup.py
--- Python-3.5.3/setup.py	2017-02-23 19:24:35.000000000 +0000
+++ Python-3.5.3.tpg/setup.py	2017-02-23 19:25:00.843618210 +0000
@@ -1999,7 +1999,7 @@
             with open(ffi_h) as f:
                 for line in f:
                     line = line.strip()
-                    if line.startswith(('#define LIBFFI_H',
+                    if line.startswith(('#define _MULTIARCH_HEADER',
                                         '#define ffi_wrapper_h')):
                         break
                 else:
