--- Python-3.7.0a4/setup.py.link~	2018-01-09 04:05:14.000000000 +0100
+++ Python-3.7.0a4/setup.py	2018-01-31 17:55:06.486014467 +0100
@@ -657,7 +657,7 @@ class PyBuildExt(build_ext):
                                libraries=['m']) )
 
         # time libraries: librt may be needed for clock_gettime()
-        time_libs = []
+        time_libs = [ 'm' ]
         lib = sysconfig.get_config_var('TIMEMODULE_LIB')
         if lib:
             time_libs.append(lib)
@@ -2060,7 +2060,7 @@ class PyBuildExt(build_ext):
         undef_macros = []
         if '--with-system-libmpdec' in sysconfig.get_config_var("CONFIG_ARGS"):
             include_dirs = []
-            libraries = [':libmpdec.so.2']
+            libraries = [':libmpdec.so.2', 'm']
             sources = ['_decimal/_decimal.c']
             depends = ['_decimal/docstrings.h']
         else:
