--- Python-3.3.0/Lib/distutils/sysconfig.py	2012-09-29 12:00:28.000000000 +0400
+++ Python-3.3.0-1/Lib/distutils/sysconfig.py	2013-03-28 17:53:13.945246411 +0400
@@ -107,7 +107,13 @@
                 incdir = os.path.join(get_config_var('srcdir'), 'Include')
             return os.path.normpath(incdir)
         python_dir = 'python' + get_python_version() + build_flags
-        return os.path.join(prefix, "include", python_dir)
+        multiarch = ''
+        if plat_specific:
+            arch = os.uname()[4]
+            if arch.endswith('86'):
+                arch = 'i386'
+            multiarch = 'multiarch-%s-%s' % (arch, sys.platform)
+        return os.path.join(prefix, "include", multiarch, python_dir)
     elif os.name == "nt":
         return os.path.join(prefix, "include")
     elif os.name == "os2":
