--- Python-3.7.0b1/setup.py.omv~	2018-02-02 16:52:59.053044214 +0100
+++ Python-3.7.0b1/setup.py	2018-02-02 16:54:12.506371770 +0100
@@ -1967,7 +1967,7 @@ class PyBuildExt(build_ext):
                         include_dirs=include_dirs,
                         extra_compile_args=extra_compile_args,
                         extra_link_args=extra_link_args,
-                        libraries=[],
+                        libraries=['dl'],
                         sources=sources,
                         depends=depends)
         # function my_sqrt() needs libm for sqrt()
