diff -Naur Python-3.5.3/setup.py Python-3.5.3.tpg/setup.py
--- Python-3.5.3/setup.py	2017-01-17 07:57:54.000000000 +0000
+++ Python-3.5.3.tpg/setup.py	2017-02-23 19:06:18.022036334 +0000
@@ -615,7 +615,7 @@
                                libraries=math_libs) )
 
         # time libraries: librt may be needed for clock_gettime()
-        time_libs = []
+        time_libs = ['m']
         lib = sysconfig.get_config_var('TIMEMODULE_LIB')
         if lib:
             time_libs.append(lib)
@@ -1974,7 +1974,7 @@
                         include_dirs=include_dirs,
                         extra_compile_args=extra_compile_args,
                         extra_link_args=extra_link_args,
-                        libraries=[],
+                        libraries=['m', 'dl'],
                         sources=sources,
                         depends=depends)
         # function my_sqrt() needs math library for sqrt()
@@ -2032,7 +2032,7 @@
                                                          'Modules',
                                                          '_decimal',
                                                          'libmpdec'))]
-            libraries = []
+            libraries = ['m']
             sources = [
               '_decimal/_decimal.c',
               '_decimal/libmpdec/basearith.c',
