diff -up Python-3.7.0b1/Include/pyatomic.h.omv~ Python-3.7.0b1/Include/pyatomic.h
--- Python-3.7.0b1/Include/pyatomic.h.omv~	2018-03-15 20:27:32.422760692 +0000
+++ Python-3.7.0b1/Include/pyatomic.h	2018-03-15 20:28:11.383840092 +0000
@@ -6,9 +6,15 @@
 
 #include "pyconfig.h"
 
+#ifdef __cplusplus
+#include <atomic>
+#define _Atomic(T) atomic<T>
+using namespace std;
+#else
 #if defined(HAVE_STD_ATOMIC)
 #include <stdatomic.h>
 #endif
+#endif
 
 
 #if defined(_MSC_VER)
diff -up Python-3.7.0b1/Programs/python.c.omv~ Python-3.7.0b1/Programs/python.c
